Sources/UI/SwiftUI/View Modifiers/PhotoPreviewModifier.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  PhotoPreviewModifier.swift
+//  PovioKit
+//
+//  Created by Ndriqim Nagavci on 14/08/2024.
+//  Copyright © 2024 Povio Inc. All rights reserved.
+//
+
+import SwiftUI
+
+@available(iOS 14.0, *)
```

Comment:
We are not supporting SwiftUI for versions below 15. Event 15 is questionable since we don't want to build apps for iOS 15 in swiftUI. Let's bump to 15 for now. 
But if you feel there are some constraints for using even 15, lmk.